### PR TITLE
patch for conversion revenue

### PR DESCRIPTION
**Pull request by KennyHuRadar**

### CODE REVIEW

File: ios/RNRadar.m [hunk 1]
```diff
@@ -1152,9 +1152,9 @@ - (void)didUpdateToken:(RadarVerifiedLocationToken *)token  {
     };
     
     if (revenue) {
-        [Radar logConversionWithName:name metadata:metadata completionHandler:completionHandler];
-    } else {
         [Radar logConversionWithName:name revenue:revenue metadata:metadata completionHandler:completionHandler];
+    } else {
```
Comment (lmeier) [inline ios/RNRadar.m:1156]: When we're parsing revenue, let's do this:
```
    NSNumber *revenue = nil;
    if (revenueObj != nil) {
        if ([revenueObj isKindOfClass:[NSNumber class]]) {
            revenue = (NSNumber *)revenueObj;
        } else if ([revenueObj isKindOfClass:[NSString class]]) {
            revenue = @([(NSString *)revenueObj doubleValue]);
        }
    }
  ```

and then change `if (revenue) {` to `if (revenue != nil) {`

**Review by lmeier — CHANGES_REQUESTED** [review]

File: ios/RNRadar.m (hunk 1 above)
Comment (KennyHuRadar) [reply ios/RNRadar.m:1156]: we don't need to handle the case where it is a string as the ts is types as number.

Comment (KennyHuRadar) [reply ios/RNRadar.m:1156]: image
image

Dogfooding to ensure that the current implementation handles having revenue defined as param or missing in param

**Review by lmeier — APPROVED** [review]
let's bump native SDKs as well, then this is good to go

**Pull request merged**